### PR TITLE
AUSTA-273 Removed references to Linux work directories

### LINKED ISSUE #9: Blueprint "Autounattend Windows 10 (Win10) BIOS Single ISO on Windows Worker for ESXi" references "Automation Worker Linux Base Directory".
The blueprint `Autounattend Windows 10 (Win10) BIOS Single ISO on Windows Worker for ESXi` references `Automation Worker Linux Base Directory`.
image

### PR DESCRIPTION
**Pull request by h4xhor**
AUSTA-273: Removed references to Linux work directories in `Copy Drivers ISO to ESXi Host with Windows Worker` and `Load Drivers ISO into VM with Windows Worker`.

closes [https://github.com/Attune-Automation/Automate-Windows-Installation-with-autounattend-on-ESXi/issues/9](https://github.com/Attune-Automation/Automate-Windows-Installation-with-autounattend-on-ESXi/issues/9).

### CODE REVIEW

File: steps/setbootorderwithwindowsworker/script.txt
Comment (brentonford) [inline steps/setbootorderwithwindowsworker/script.txt:1]: @h4xhor, fix this script, it wasn't merged properly.

**Review by brentonford — CHANGES_REQUESTED** [review]
@h4xhor, there's an issue with a script that needs to be corrected

File: steps/setbootorderwithwindowsworker/script.txt
Comment (jarrodchesney) [reply steps/setbootorderwithwindowsworker/script.txt:1]: The file still has merge conflict lines in it, like:
```
<<<<<<<< HEAD:steps/setbootorderwithwindowsworker/script.txt
```

Comment (h4xhor) [reply steps/setbootorderwithwindowsworker/script.txt:1]: Hi @brentonford and @jarrodchesney . I have fixed the merge conflicts.

**Pull request merged**